.rubocop.yml

```diff
@@ -70,3 +75,8 @@ RSpec/FilePath:
 
 RSpec/InstanceVariable:
   Enabled: false
+
+RSpec/DescribedClass:
+  Exclude:
+    - 'spec/hydra/works/models/concerns/generic_file/characterization_spec.rb'
```

Comment:
Rubocop will complain about not using described_class in the spec.  However, using described_class doesn't permit this section of the spec to run:
```ruby
describe Hydra::Works::GenericFile::Characterization do
  let(:demo_class) do
    Class.new(Hydra::Works::GenericFile::Base) do
      include Hydra::Works::GenericFile::Characterization
    end
  end
```